src/robusta/model/playbook_definition.py

```diff
@@ -1,44 +1,40 @@
 import hashlib
+import math
+import json
+from os import name
 from typing import Any, Dict, List
-
-from pydantic import BaseModel, PrivateAttr
+from pydantic import BaseModel, PrivateAttr, validator
 
 from robusta.core.playbooks.trigger import Trigger
 from robusta.model.playbook_action import PlaybookAction
 
 
 class PlaybookDefinition(BaseModel):
-    sinks: List[str] = None
+    name: str
+    sinks: List[str] = None  # compatibility
     triggers: List[Trigger]
-    actions: List[Dict[str, Any]]
-    _actions: List[PlaybookAction] = PrivateAttr()
-    stop: bool = False
-    _playbook_id: str = PrivateAttr()
+    actions: List[PlaybookAction] = []
+    tags: Dict[str, str] = {}
+    priority: float = math.inf
+    halt: bool = False
+    disabled: bool = False
+
 
-    def __init__(self, **data: Any):
-        super().__init__(**data)
-        self._actions = []
-        for action in self.actions:
-            if len(action.keys()) != 1:
-                raise Exception(f"Action must have a single name: {action.keys()}")
+    # TODO: better to be in root_validator of the PlaybookAction
+    @validator('actions', pre=True, each_item=True)
+    def create_action(cls, action_data):
```

Comment:
I think this is a little confusing.
We call it for each list item, but get the list each time, and assume the iterator is on the same position.

Would it be simpler to run once, and validate the whole list?

Reply:
Not sure I got you. The each_item flag just saves me from writing an extra loop. The action data is a dict object